server/src/main/java/org/opensearch/indices/DefaultCompositeIndexSettings.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.index.compositeindex.CompositeIndexSettings;
+
+/**
+ * Utility to provide a {@link CompositeIndexSettings} instance containing all defaults
+ */
+public class DefaultCompositeIndexSettings {
```

Comment:
Do we need this to be a separate class, or can this be within `CompositeIndexSettings` itself?